contracts/zkdvrf.sol

```diff
@@ -146,24 +146,25 @@ contract zkdvrf is Ownable {
         require(contractPhase == Status.Ready, "Contract not ready");
 
         if (currentRoundNum != 0) {
-            require(roundToRandom[currentRoundNum] != bytes32(0), "Earlier round not completed");
+            require(roundToRandom[currentRoundNum].value != bytes32(0), "Earlier round not completed");
         }
 
         currentRoundNum++;
         uint256 currentTimestamp = block.timestamp;
         roundInput[currentRoundNum] = currentTimestamp.toString();
     }
 
-    function submitPartialEval(Pairing.G1Point memory pEval, IPseudoRand.PartialEvalProof memory proof) public {
+    function submitPartialEval(IPseudoRand.PartialEval memory pEval) public {
```

Comment:
is storing the proof on the contract useful?

Reply:
In the future, if we implement lazy verification, the proof won't be verified immediately when it is sent onchain.